### PR TITLE
MINOR: Update documentation for 0.10.1 release

### PR DESCRIPTION
**Pull request by hachikuji**

### CODE REVIEW

File: docs/documentation.html [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
 <!--#include virtual="../includes/header.html" -->
 
-<h1>Kafka 0.10.0 Documentation</h1>
+<h1>Kafka 0.10.1 Documentation</h1>
```
Comment (gwenshap) [inline docs/documentation.html:20]: you need a link to 0.10.0 below.

Comment (hachikuji) [reply docs/documentation.html:20]: Good call. Fixed.

**Comment by gwenshap** [comment]
LGTM

**Comment by ijuma** [comment]
We want this in trunk too, right? Also, seems like this was merged, so the PR should be closed (non-trunk PRs are not closed automatically).

**Comment by ijuma** [comment]
Shall we close this?

**Comment by guozhangwang** [comment]
Is this still valid? If not could you close this PR?

**Pull request closed without merging**